form.js

```diff
@@ -0,0 +1,20 @@
+// Function to validate and submit the form
+function validateAndSubmit() {
+  const emailInput = document.getElementById('email');
+  const errorMessage = document.getElementById('error-message');
+
+  // Check if the email is in lowercase
+  if (emailInput.value !== emailInput.value.toLowerCase()) {
+    errorMessage.innerText = 'Email should be in lowercase.';
+    return false; // Prevent form submission
```

Comment:
- [ ] Great job on implementing the validationa with the eror message showing up but the form didnt submit normally when i input the email in lower case and according to the project requirement says 
 - The contact form contains at least one validation checking if the email is written in all lowercase:

If a user submits the contact form with the email field containing uppercase characters an error message should show up.
If a user submits the contact form with the email field containing only lowercase characters the form is sent normally.
- The form `"submit"` or the submit button/input `"click"` event is used to attach the function checking the email format.